src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -407,7 +407,8 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory
   class RefactoringParameters(
     val importsToAdd: List[(String, String)] = Nil,
     val options: List[Participant] = DefaultOptions,
-    val deps: Dependencies.Value = Dependencies.RemoveUnneeded)
+    val deps: Dependencies.Value = Dependencies.RemoveUnneeded,
+    val organizeLocalImports: Boolean = true)
```

Comment:
here is PR in scala-ide to toggle feature: https://github.com/scala-ide/scala-ide/pull/1089